npm/oxlint/configuration_schema.json

```diff
@@ -2,6 +2,7 @@
   "$schema": "http://json-schema.org/draft-07/schema#",
   "title": "Oxlintrc",
   "description": "Oxlint Configuration File\n\nThis configuration is aligned with ESLint v8's configuration schema (`eslintrc.json`).\n\nUsage: `oxlint -c oxlintrc.json --import-plugin`\n\n::: danger NOTE\n\nOnly the `.json` format is supported. You can use comments in configuration files.\n\n:::\n\nExample\n\n`.oxlintrc.json`\n\n```json\n{\n\"$schema\": \"./node_modules/oxlint/configuration_schema.json\",\n\"plugins\": [\"import\", \"typescript\", \"unicorn\"],\n\"env\": {\n\"browser\": true\n},\n\"globals\": {\n\"foo\": \"readonly\"\n},\n\"settings\": {\n},\n\"rules\": {\n\"eqeqeq\": \"warn\",\n\"import/no-cycle\": \"error\",\n\"react/self-closing-comp\": [\"error\", { \"html\": false }]\n},\n\"overrides\": [\n{\n\"files\": [\"*.test.ts\", \"*.spec.ts\"],\n\"rules\": {\n\"@typescript-eslint/no-explicit-any\": \"off\"\n}\n}\n]\n}\n```",
+  "allowComments": true,
```

Comment:
This is a generated file, we unfortunately will need to do this a different way otherwise it will get overridden next time someone regenerates it.

Reply:
Ah, oops, I'll check if I can make that change in the generation process as well.

Reply:
I made a ticket on the website repo to improve the docs for this https://github.com/oxc-project/oxc-project.github.io/issues/598

I believe the simplest way to regenerate the configuration_schema.json is to run `cargo test -p oxc_linter`, if you need to test your changes. It uses the schemars crate to pull information/types from rust doc comments and decorators across the oxlint code.

Reply:
nice, thanks!